modules/platform/nuxeo-easyshare-core/src/main/resources/i18n/messages.properties

```diff
@@ -1,10 +1,18 @@
 easyshare.label.title=Nuxeo - Easyshare
 easyshare.label.description=Nuxeo Easyshare Folder
-easyshare.label.denied=Oh no! This folder doesn't exist.
-easyshare.label.expired=Oh no! This Easy Share folder has expired.
+easyshare.label.denied=Oh no! This access is secured.
```

Comment:
I think we want to keep the denied error as the folder doesn't exist, otherwise, it gives extra information to the user making the request (ie: the document with this id/path exists).

Reply:
OK, I can keep the logic in the code but just change the label ? Could you validate this part internally ?